infrastructure/deployment/deploy.sh

```diff
@@ -268,6 +268,11 @@ split_and_join() {
    SPLIT=$(echo $text | sed -e "s/$separator_for_splitting/$separator_for_joining/g")
    echo $SPLIT
 }
+cleanup_docker_images()
+{
+   echo "Cleaning up the docker images"
+   /usr/bin/docker system prune -af | sudo tee -a /var/log/docker-prune.log > /dev/null
+}
```

Comment:
I really like these functions 👍

Reply:
@rikukissa Wouldn't it be better to not remove the dependencies? Because it will probably increase the deploy time by quite a bit otherwise

Reply:
I think those would only be removed if they are currently not being used right? But they would assuming there is a stack running currently

Reply:
Okay doesn't matter as we will be downloading it after the deploy anyway